src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -222,8 +218,8 @@ class ActivityFeed extends React.Component<Props, State> {
                             currentUser={currentUser}
                             onTaskAssignmentUpdate={this.updateTaskAssignment}
                             onCommentDelete={hasCommentPermission ? this.deleteComment : noop}
-                            onTaskDelete={hasTaskPermission ? this.deleteTask : noop}
-                            onTaskEdit={hasTaskPermission ? this.updateTask : noop}
+                            onTaskDelete={hasCommentPermission ? this.deleteTask : noop}
```

Comment:
Can you add a comment here explaining this behavior? Tasks is technically a separate permission but we don't have access to it

Reply:
will do